tests/commands/build.spec.ts

```diff
@@ -193,16 +193,19 @@ describe('balena build', function () {
 		}
 		const arch = 'rpi';
 		const deviceType = 'raspberry-pi';
-		const fsModPath = 'mz/fs';
+		const fsModPath = 'fs';
 		const fsMod = await import(fsModPath);
 		const qemuModPath = '../../build/utils/qemu';
 		const qemuMod = require(qemuModPath);
 		const qemuBinPath = await qemuMod.getQemuPath(arch);
 		try {
 			mock(fsModPath, {
 				...fsMod,
-				exists: async (p: string) =>
-					p === qemuBinPath ? true : fsMod.exists(p),
+				promises: {
+					...fsMod.promises,
+					access: async (p: string) =>
+						p === qemuBinPath ? undefined : fsMod.promises.exists(p),
```

Comment:
Shouldn't this line call `access` instead of `exists`? -
`p === qemuBinPath ? undefined : fsMod.promises.access(p),`

Reply:
You're right, I guess we never hit the real version during tests but it should be fixed

Reply:
fixed